packages/keyring-eth-ledger-bridge/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/eth-ledger-bridge-keyring",
-  "version": "10.0.0",
+  "version": "10.1.0",
```

We have a major breaking change, this should not be a minor:

```suggestion
  "version": "11.0.0",
```